apps/participants/utils.py

```diff
@@ -33,6 +33,15 @@ def get_participant_team_id_of_user_for_a_challenge(user, challenge_id):
     return None
 
 
+def get_participant_team_name_of_user_for_a_challenge(user, challenge_id):
+    """Returns the participant team object for a particular user for a particular challenge"""
+    participant_team_id = get_participant_team_id_of_user_for_a_challenge(user, challenge_id)
+    if participant_team_id:
+        return ParticipantTeam.objects.get(pk=participant_team_id).team_name
+    else:
```

Comment:
Please remove this condition and simply use `return`

Reply:
@RishabhJain2018 
As we are getting `participant_team_id` from `get_participant_team_id_of_user_for_a_challenge`, there is a case where we might get where `participant_team_id = None`.